Update manage-projects.md (#2116)
semgrep/semgrep-docs: docs/deployment/manage-projects.md

- **Duration**: The amount of time the scan took to complete in hours or minutes

:::note

> Scan details, such as logs, are available for scans run in the past **1 month**. The Semgrep UI does not display scans that are older than 30 days. Expanding it significantly would introduce performance issues due to the increased volume of stored scan data.

Reviewer: ```suggestion
Scan details, such as logs, are available for scans run in the past **1 month**. Semgrep AppSec Platform does not display scan details older than 30 days, since this introduces performance issues due to the increased volume of stored scan data.
```
